arithmetization/src/main/java/net/consensys/linea/zktracer/module/mmu/MmuOperation.java

```diff
@@ -88,12 +86,8 @@ protected int computeLineCount() {
         : 0;
   }
 
-  public int computeMmioLineCount() {
-    if (mmioLineCount != -1) {
-      return mmioLineCount;
-    }
-
-    mmioLineCount = 0;
+  public int mmioLineCount() {
+    int mmioLineCount = 0;
     if (traceMe()) {
       for (int i = 0; i < mmuData().numberMmioInstructions(); i++) {
         mmioLineCount +=
```

Comment:
In the function body that follows we use the method `numberOfRowOfMmioInstruction`. It would be great to rename it using the word `Line` instead of `Row` e.g. `lineCountOfMmioInstruction`.

Reply:
done